Client/Frontend/Browser/PasswordManager.swift

```diff
@@ -1,6 +1,6 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Nit - I think a better name for this class would be `PasswordHelper` and then the associated JS file can be `PasswordHelper.js`